frontend/src/components/TracingIntegration/TracesFetcher.ts

```diff
@@ -72,6 +73,20 @@ export class TracesFetcher {
             'Loading traces for all clusters. Tracing is not configured to store traces per cluster.'
           );
         }
+        const firstTraceTimestamp = Math.min(...traces.map(s => s.startTime));
+        // If the traces cover less than 90% of the selected period, show an info message
+        // The order in which traces are returned is not deterministic
+        const endDate = q.endMicros ? q.endMicros : Date.now() * 1000;
+        const timeRange = (endDate - q.startMicros) * 0.9;
+
+        if (firstTraceTimestamp && this.lastFetchMicros && this.lastFetchMicros - firstTraceTimestamp < timeRange) {
+          this.onInfo(
+            `The ${q.limit} traces do not fully cover the time period. You may want to increase the limit, reduce the time range, or define a custom time range.`
```

Comment:
After using this, I think it would be nice if the message indicated the %.  Could you change things so that we could say

`The ${q.limit} traces cover only ${tracePercent}% of the time period. You may want to increase the fetch limit, reduce the time range, or define a custom time range.`

Reply:
Updated: 
![image](https://github.com/user-attachments/assets/ca61aa36-94b8-4da3-993e-7c0082eb7b0b)